stories/Components/UIcomponents/Cards/ContentCard/contentcard.scss

```diff
@@ -165,6 +172,16 @@
       margin-left: 0;
     }
   }
+
+  $colors: ("yellow": $color-yellow, "red": $color-red, "green": $color-green, "blue": $color-azure);
+    @each $name, $color in $colors {
```

Comment:
@Ashish-srij Fix alignment here. Needs to be 2 spaces.

Reply:
@shishirphadke formated.